record-two/index.html.in

```diff
@@ -9,6 +9,15 @@
     <navbar></navbar>
 
     <div class="container-fluid">
+        <div id="record2-bookmark-container" class="col-xs-12 meta-icons" ng-controller="RecordController as ctrl">
+            <a id="create-record" ng-if="reference.canCreate && ctrl.editRecord" title="Click here to create a record" ng-click="ctrl.createRecord()">
+                <span class="glyphicon glyphicon-plus"></span> <strong>Create Record</strong>
+            </a>&nbsp;&nbsp;
+            <a id="edit-record" ng-if="reference.canUpdate && ctrl.editRecord" title="Click here to edit this record" ng-click="ctrl.editRecord()">
+                <span class="glyphicon glyphicon-pencil"></span> <strong>Edit Record</strong>
+            </a>
+        </div>
+
```

Comment:
I don't expect any of the expressions inside the `ng-if`s and `ng-clicks` to change after first digest; perhaps we can benefit from some one-time binding here.